harness/determined/core/_train.py

```diff
@@ -67,11 +68,12 @@ def _get_last_validation(self) -> Optional[int]:
         logger.debug(f"_get_last_validation() -> {steps_completed}")
         return steps_completed
 
-    def report_training_metrics(
+    def report_trial_metrics(
         self,
-        steps_completed: int,
-        metrics: Dict[str, Any],
-        batch_metrics: Optional[List[Dict[str, Any]]] = None,
+        metric_type: str,
```

Comment:
nit: what about naming this something more generic like `series` or just `metric_name`? `type` implies to me that there's an explicit limited set of "types" allowed

Reply:
metric_name I think could get confused with what you would nest insdie a metric package eg `loss`
serires could work too. I went with metric_type becuase it maps directly to the apis but we could always deviate if something is more helpful to the user

Reply:
completely subjective, but `type` just makes me think "what types are supported/exist?" when really here the user is just assigning a label of sorts to a metric. 

"series" seems like a good fit to me, but i'm not bent on it. keeping `metric_type` consistent with the underlying REST APIs makes sense too.

Reply:
have the rest apis landed to main yet?  I think we should change the name from `metric_type` to `series`, because I think that name is _way_ better for the concept here.

Reply:
yes, they have but I haven't published the release notes yet.
series does sound good 🤔

Reply:
is the plan to change this to "series"? if so, should we wait for backend API name change? since if we land this, then change it later, it could become a user-facing change.

Reply:
slack conversation webui,

Reply:
Personally, I don't see any issue with `metric_type` for what it's worth. The PR to make this `group` is here: https://github.com/determined-ai/determined/pull/7353/files but honestly I would rather just let it be. Could we look into that PR @azhou-determined @rb-determined-ai ?